src/gui/app/directives/controls/sort-tags-row.js

```diff
@@ -47,14 +48,27 @@
                                 <input type="checkbox" ng-click="$ctrl.toggleSortTag(tag)" ng-checked="$ctrl.item.sortTags.includes(tag.id)">
                                 <div class="control__indicator"></div>
                             </label>
-                            <div ng-if="sts.getSortTags($ctrl.context).length === 0">No tags created yet</div>
+                            <div class="button mb-2" ng-click="editSortTags()" ng-if="sts.getSortTags($ctrl.context).length === 0">
+                                Add tags
+                            </div>
+                            <hr class="divider mt-1 mb-1" ng-if="sts.getSortTags($ctrl.context).length > 0" />
+                            <div class="button mt-4 mb-2" ng-click="editSortTags()" ng-if="sts.getSortTags($ctrl.context).length > 0">
+                                Add/edit tags
+                            </div>
                         </div>
                     </script>
-                <div>
+                </div>
```

Comment:
psst. This one's worth an extra comment. I thought I had messed something up when I looked at the diff, so now you get to read this text and waste ten seconds.